Yoast/wordpress-seo
# Added a check to make sure the required tables are accessible

## Summary

This PR can be summarized in the following changelog entry:

* Makes sure a the required tables are accessible.

## Relevant technical choices:

* Added the checks for the public methods.

## Test instructions

This PR can be tested by following these steps:

1. Checkout `trunk`
1. Rename the `wp_yoast_links` table to something else or remove it entirely
1. Move a post to the trash and remove and then delete it permanently
1. In your vagrant directory there is an error log. Probably `VVV/log/php7.0_errors.log`
1. See some SQL errors added to the log.
1. Checkout this branch. Yay
1. Follow step 3 and 4. It should be fixed now.
